test/Compilation/Positive.hs

```diff
@@ -238,5 +238,10 @@ tests =
       "Applications with lets and cases in function position"
       $(mkRelDir ".")
       $(mkRelFile "test037.juvix")
-      $(mkRelFile "out/test037.out")
+      $(mkRelFile "out/test037.out"),
+    posTest
+      "Match with more arguments than patterns"
+      $(mkRelDir ".")
```

Comment:
The evaluator doesn't currently run on these tests. (The match to case compilation PR will enable them). Did you test this manually?

Reply:
I tested it in the repl